src/modules/PrimarySection/PrimarySection.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import Container from '../../UI/Container';
+import Header from '../Header';
+
+const PrimarySection = () => {
+  return (
+    <section className={styles.primarySection}>
+      <div className={styles.backgroundContainer}>
+        <Header />
+        <Container>
+          <div className={styles.content}>
+            <h1 className={styles.title}>Book pool time and amenities at luxury hotels</h1>
+            <form action="#" className="header__search-form search-form">
+              <div className="search-from__filter">
+                <ul className="search-from__filter-list">
+                  <li className="search-from__filter-item search-from__filter-item_active">All</li>
```

Comment:
please reside it to an array and loop through it

Reply:
Yeah, I was going to do it exactly like this. I just haven't touched it yet, just did copy/paste from initial.